packages/gtk_doc.rb

```diff
@@ -31,14 +31,17 @@ class Gtk_doc < Package
   depends_on 'docbook_xsl'
 
   def self.build
-    system 'pip3 install six'
-    system 'pip install six'
-    system './autogen.sh --with-xml-catalog=/usr/local/etc/xml/catalog'
-    system './configure --with-xml-catalog=/usr/local/etc/xml/catalog'
+    system "./autogen.sh"
+    system "./configure --with-xml-catalog=/usr/local/etc/xml/catalog --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
```

Comment:
```
- --with-xml-catalog=/usr/local/etc/xml/catalog
+ --with-xml-catalog=#{CREW_PREFIX}/etc/xml/catalog
```
You should get in the habit of replacing all hardcoded '/usr/local`s with `#{CREW_PREFIX}` instead.

Reply:
I didn't do it.

Reply:
Got it anyway.